community/CONTRIBUTING.md

```diff
@@ -5,6 +5,192 @@ Thanks for considering contributing to OPEA project. The contribution process is
 
 ## All The Ways To Contribute
 
+### Contribute a GenAI Component
+
+1. Navigate to [OPEA GenAIComps](https://github.com/opea-project/GenAIComps) and locate the component folder your integration belongs to. If the microservice type already exists, review the [OPEA microservice API](https://opea-project.github.io/0.9.99/developer-guides/OPEA_API.html#opea-micro-service-api) and follow it in your implementation. Otherwise, if you are contributing a brand new microservice type, you need to define and contribute the API specifications. Please start by submit an RFC to get contributors feedback. 
```

Comment:
0.9.99 was a testing "release" of the documentation and is gone now... refer to the /latest/ version instead.
```suggestion
1. Navigate to [OPEA GenAIComps](https://github.com/opea-project/GenAIComps) and locate the component folder your integration belongs to. If the microservice type already exists, review the [OPEA microservice API](https://opea-project.github.io/latest/developer-guides/OPEA_API.html#opea-micro-service-api) and follow it in your implementation. Otherwise, if you are contributing a brand new microservice type, you need to define and contribute the API specifications. Please start by submit an RFC to get contributors feedback. 
```

Reply:
fixed